cli/commands/asset/add.go

```diff
@@ -0,0 +1,175 @@
+package asset
+
+import (
+	"bytes"
+	"encoding/json"
+	"errors"
+	"fmt"
+	"os"
+	"sort"
+	"strings"
+
+	"github.com/sensu/sensu-go/bonsai"
+	"github.com/sensu/sensu-go/cli"
+	"github.com/spf13/cobra"
+
+	goversion "github.com/hashicorp/go-version"
+	corev2 "github.com/sensu/sensu-go/api/core/v2"
+	"github.com/sensu/sensu-go/cli/commands/create"
+	"github.com/sensu/sensu-go/types"
+)
+
+var rename string
```

Comment:
Whats this lil guy doing here?

Reply:
This is the variable that the `--rename` flag value is stored in: https://github.com/sensu/sensu-go/blob/bf669448d2a82038ed885a9445e21c77824421f5/cli/commands/asset/add.go#L32.

Reply:
Wondering why its a global variable vs a regular one

Reply:
This is the pattern that the Cobra documentation shows. I believe our other command packages handle flags differently as they're coupled with our Viper config. If you know of a better way of handling this, please share!